### ISSUE TITLE
CombatFlite material

### ISSUE THREAD

**Issue by Gabykhan** [132nd-vWing/TRMA#37]
Since CombatFlite is quite abandonned, we need to build our own material in order to keep using this amazing software as our planning tool.

There are 2 options:
1 - create a new DCS theatre in CombatFlite. It's by far the cleanest and most professinal option. I did not manage to do it myself, and it might be necessary to dug into the .exe file or something like that, and it's beyond my skills. If someone feels he's able to look into it, please do so.
2 - modify an existing theatre. It's the road I took at this point. I don't use and don't plan to use the "DCS Channel" map, so I will modifiy all that's related to this.

Now appart from this initial decision, there are several files that need to be created/modified for our theater:
1 - In CombatFlite\Data
     1.1 - The theaters.xml file. It defines the main features of your theatre (like coordinates, default BE location, etc...)
     1.2 - The thechannel.xml file (because I decided to modify the "DCS Channel" theatre. It regroups all the Navpoints, Beacons and Airfields in use on the theatre. This one needs the most work to make it comprehensive and reflect DCS data.
2 - In CombatFlite\Data\DCS The Channel (again because I chose to modify the "DCS Channel" theatre)
     2.1 - Create any map in mbtile format. **I'm working on it, and I'll be able to release the first maps soon (ONC and TPC)**.

So basically the work that needs to be done here is:
1 - Find and share high quality aeronautical charts or any map that you'd like to use in CombatFlite so that I can turn them into the correct format.
2 - Add more data to thechannel.xml database to make it as comprehensive and close to DCS as possible.
3 - Dig into CombatFlite program to find out how to create a new theatre from scratch. Good thing is that all that will be done on steps 1 and 2 will be directly usable for this new theatre.

The attached .zip file contains the modified theaters.xml (with correct data) and thechannel.xml (that needs to be worked on) files.
[CombatFlite.zip](https://github.com/user-attachments/files/16736206/CombatFlite.zip)

**Comment by 132ndNeck** [comment]
Great initative!
This is above my technical know-how, but I very much support the idea, and would love for us to have a working CombatFlite file for the TRMA.

**Comment by Gabykhan** [comment]
Second package: [here you'll find the SRTM files](https://cloud.132virtualwing.org/s/y8ZTKc2CbyMctjc) to individually copy in your CombatFlite\SrtmDataFiles folder.

**Comment by Gabykhan** [comment]
Third item: the 1:1.000.000[ ONC map of the region](https://cloud.132virtualwing.org/s/5eDZ4s8d4stLqco) to place in CombatFlite\Data\DCS The Channel (again because I chose to modify the "DCS Channel" theatre).

**Comment by Gabykhan** [comment]
Fourth item: a[ topographic map of the region](https://cloud.132virtualwing.org/s/3CTHJRXqZGqJTQJ) to place in CombatFlite\Data\DCS The Channel (again because I chose to modify the "DCS Channel" theatre).

![image](https://github.com/user-attachments/assets/74dc7404-7135-42df-a59f-0490d0665ea9)

**Comment by Gabykhan** [comment]
Fidth item: a[ 1:500.000 TPC map of the region](https://cloud.132virtualwing.org/s/WdYZ88db4NPBq6b) to place in CombatFlite\Data\DCS The Channel (again because I chose to modify the "DCS Channel" theatre).

![TPC for Github](https://github.com/user-attachments/assets/ffd20109-4999-45b3-bea9-88812b245dbf)

**Comment by Gabykhan** [comment]
Last addition for the moment: a[ modern 1:500.000 map of the region](https://cloud.132virtualwing.org/s/YPtHkoEEBzNE8FC) to place in CombatFlite\Data\DCS The Channel (again because I chose to modify the "DCS Channel" theatre).

![image](https://github.com/user-attachments/assets/6f5edf78-7aed-4aa0-bd31-dbdfd1171fd2)

**Comment by 132ndNeck** [comment]
From @martinco : 

XML file with airports: https://cloud.132virtualwing.org/s/pYM4L93TqizJNL8

DCS Map: https://mdc.132virtualwing.org/files/combatflite/DCS-kola-6-12-png.mbtiles

**Comment by Gabykhan** [comment]
[ONC map has been updated](https://cloud.132virtualwing.org/s/miZocjCzpC4sEbP). Others will follow.

**Comment by RedDog132nd** [comment]
May I request a bit of help on this? I installed all files as per the above procedure and CF works relatively well using the channel for Kola
But I do not see the different map? I see yet another (default?) map. I'm particularily interested in martin's F10 map but although it is in the data folder, I have no idea how to display or activate it in Combatflight.
Sorry for asking, I know it's not the place for support on these things, so feel free to redirect me if necessary
thanks

**Comment by Gabykhan** [comment]
Followinf=g DCS update from 30/10/2024 the following airbases have been added:

> DCS: Kola Map by orbx
> 
> New:
> 
>     Added ENAN Andoya Airfield
>     Added RU59 Alakourtti Airfield

You can therefore replace your .xml file in CombatFlite with the attached one (see first post for more details).

[thechannel.zip](https://github.com/user-attachments/files/17713519/thechannel.zip)

**Comment by jonde99** [comment]
I noticed that Alternate WP always fixes the something in channel map. Any way tonfix that?

**Comment by RedDog132nd** [comment]
Question:
On the CF file for TRMA I see the ILS symbols for Bodo RWY 07 and RWY 25
but I cannot find these on the layers.
I thought they were part of the map, but when I change maps, they remain displayed. 

If the ILS is ok for RWY 07, the ILS for RWY 25 needs to offset
That's what I'm trying to do.

Any hints on how to?
Thanks

**Comment by 132ndNeck** [comment]
> Question: On the CF file for TRMA I see the ILS symbols for Bodo RWY 07 and RWY 25 but I cannot find these on the layers. I thought they were part of the map, but when I change maps, they remain displayed.
> 
> If the ILS is ok for RWY 07, the ILS for RWY 25 needs to offset That's what I'm trying to do.
> 
> Any hints on how to? Thanks

![image](https://github.com/user-attachments/assets/1796efee-5c68-40b9-989f-e50a94f6d7f2)
Is it this you are referring to? Then just send me the coordinate where the RWY 25 point should be moved to, and I can update it

**Comment by RedDog132nd** [comment]
No, the coordinates are OK, the track is not. It should be offset to the south
![image](https://github.com/user-attachments/assets/5496c817-6113-4011-9b90-b4357d7c126f)

**Comment by 132ndNeck** [comment]
> No, the coordinates are OK, the track is not. It should be offset to the south ![image](https://private-user-images.githubusercontent.com/171424165/398876371-5496c817-6113-4011-9b90-b4357d7c126f.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzUzMDE1NTcsIm5iZiI6MTczNTMwMTI1NywicGF0aCI6Ii8xNzE0MjQxNjUvMzk4ODc2MzcxLTU0OTZjODE3LTYxMTMtNDAxMS05YjkwLWI0MzU3ZDdjMTI2Zi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMjI3JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTIyN1QxMjA3MzdaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1iMzEyYzFjYmVmMjdkNzc2Y2E4Mzc3ZTRmMzIzZjk3ZTYxODc0ZDUxMWZhNDRlN2EzMjc0YWUxMjc2NDRiMTE2JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.AB9USAO-NCyeEQGPx8z6k-JwVOd_gHEGBwuX3-IIb7o)
> 
> I think the latest update this week fixed the ILS offset in the mission editor

Ah, now I get it. Ok, I will need to check. Not sure where they are edited.

**Comment by 132ndNeck** [comment]
Answer from Martin:

![image](https://github.com/user-attachments/assets/4ee2e6a8-e6b4-4d9e-85f9-649629027829)

**Comment by RedDog132nd** [comment]
Yep, saw it. Made a few test and we should change only the magcourse to 266

```
<ILS>
        <RWY>25</RWY>
        <ID>BD</ID>
        <ChFrq>108.7</ChFrq>
        <MagCrs>266</MagCrs>
        <TDZLat>67.271874</TDZLat>
        <TDZLon>14.397420</TDZLon>
      </ILS>
```

**Comment by 132ndNeck** [comment]
Do you have the data for RWY 07 aswell? Then I can uppdate the file and make it available here

**Comment by RedDog132nd** [comment]
no change on RWY 07

**Comment by 132ndNeck** [comment]
> no change on RWY 07

I do not have the data for RWY 07. If you are able to share what you have for RWY07, can upload the updated file here

**Comment by RedDog132nd** [comment]
rgr

Slight correction for RWY 25: 268°

```
<ILS>
        <RWY>25</RWY>
        <ID>BD</ID>
        <ChFrq>108.7</ChFrq>
        <MagCrs>268</MagCrs>
        <TDZLat>67.271874</TDZLat>
        <TDZLon>14.397420</TDZLon>
      </ILS>
```
 and RWY 07
```
<ILS>
        <RWY>07</RWY>
        <ID>BO</ID>
        <ChFrq>110.3</ChFrq>
        <MagCrs>072</MagCrs>
        <TDZLat>67.266601</TDZLat>
        <TDZLon>14.334084</TDZLon>
      </ILS>
```

But this one was in the master file already - I do think that the last one uploaded above is not complete, don't use it

**Comment by Gabykhan** [comment]
New map for CombatFlite has been created: [Skyvector for Kola](https://cloud.132virtualwing.org/s/Y4oPQDJ4KF4A3HJ). Enjoy!

![Image](https://github.com/user-attachments/assets/a17a2a8e-31bb-44b3-9055-f46bd6287277)

**Comment by Shadoga** [comment]
https://github.com/132nd-vWing/TRMA/issues/37#issuecomment-2332519876
The 2 download link in the above post leading to strategic-dcs.com are dead.

**Comment by martinco** [comment]
> [#37 (comment)](https://github.com/132nd-vWing/TRMA/issues/37#issuecomment-2332519876) The 2 download link in the above post leading to strategic-dcs.com are dead.

https://mdc.132virtualwing.org/files/combatflite/DCS-kola-6-12-png.mbtiles
